profiles/seccomp/default.json

```diff
@@ -746,6 +746,23 @@
 				]
 			},
 			"excludes": {}
+		},
+		{
+			"names": [
+				"get_mempolicy",
+				"mbind",
+				"name_to_handle_at",
+				"set_mempolicy"
+			],
+			"action": "SCMP_ACT_ALLOW",
+			"args": [],
+			"comment": "",
+			"includes": {
+				"caps": [
+					"CAP_SYS_NICE"
+				]
+			},
+			"excludes": {}
 		}
 	]
-}
\ No newline at end of file
+}
```

Comment:
I think you will need to keep the original file ending without a new line to get the validation to work.

Reply:
Will do.